### PR TITLE
lab proposal for HL-StarterKit

### PR DESCRIPTION
**Pull request by ravinayag**
Lab  proposal for HL Starterkit

Signed-off-by: Ravi Vasagam <ravinayag@gmail.com>

## Checklist for Hyperledger Lab Proposal PR

- [ ] Read [README.md](https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/master/README.md) carefully
- [ ] Make sure your changes are committed with the proper sign-off
- [ ] Use the labs name as the PR title
- [ ] Copy the short description of the proposal as the description of the PR
- [ ] Remove this checklist section
- [ ] Submit your PR and the labs stewards will review your proposal
- [ ] Ask your sponsor to confirm sponsorship by stating so in a comment to the PR

### CODE REVIEW

File: labs/StarterKit.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+
+# Sponsor
+
+- Name - Bobbi Muscara (bobbi@ledgeracadamy.com) - Chair of the HL-LMDWG workgroup
+
+# Pre-existing repository
+
+- https://github.com/ravinayag/HL-StartertKit.git
```
Comment (tkuhrt) [inline labs/StarterKit.md:47]: Looks like [one of the commits](https://github.com/ravinayag/HL-StartertKit/commit/82b3627d51eacb2ec1ec702ac51322609d8df82f) in this repo has not been signed off. Can you correct?

**Review by tkuhrt — COMMENTED** [review]
Overall, this looks like a good lab proposal. The only thing I would ask is if you can update the lab proposal so that Hyperledger and the project names are capitalized (e.g., instead of besu, use Besu)

File: labs/StarterKit.md (hunk 1 above)
Comment (ravinayag) [reply labs/StarterKit.md:45]: Updated with signoff's

**Comment by ravinayag** [comment]
I capitalized all project names,  Thank you

**Review by lehors — APPROVED** [review]
LGTM. The repo is no longer visible for some reason so I couldn't look at what's already there. Have you considered leveraging Minifabric?

**Review by tkuhrt — APPROVED** [review]
LGTM

**Comment by tkuhrt** [comment]
@ravinayag : I notice that the branch [trashmain](https://github.com/ravinayag/HL-StarterKit/commits/trashmain) on your repo is also missing some DCO signoffs. Can you correct and/or remove this branch and then transfer the [HL-StarterKit GitHub repo](https://github.com/ravinayag/HL-StarterKit/) to @tkuhrt, then I can transfer it to the Hyperledger Labs organization.

**Comment by tkuhrt** [comment]
https://github.com/hyperledger-labs/HL-StarterKit is now available. @ravinayag : Please accept the invite to the team for this lab.

**Comment by ravinayag** [comment]
> @ravinayag : I notice that the branch [trashmain](https://github.com/ravinayag/HL-StarterKit/commits/trashmain) on your repo is also missing some DCO signoffs. Can you correct and/or remove this branch and then transfer the [HL-StarterKit GitHub repo](https://github.com/ravinayag/HL-StarterKit/) to @tkuhrt, then I can transfer it to the Hyperledger Labs organization.

@tkuhrt : I removed that repo,(trashmain)  and  transferred it to @tkuhrt 

@lehors : Could you confirm for me,  Do you still have a problem accessing it?

**Pull request merged**